src/architecture/unixfs.md

```diff
@@ -0,0 +1,611 @@
+---
+title: UnixFS
+description: >
+  UnixFS is a Protocol Buffers-based format for describing files, directories,
+  and symlinks as DAGs in IPFS.
+date: 2022-10-10
+maturity: reliable
+editors:
+  - name: David Dias
+    github: daviddias
+    affiliation:
+      name: Protocol Labs
+      url: https://protocol.ai/
+  - name: Jeromy Johnson
+    github: whyrusleeping
+    affiliation:
+      name: Protocol Labs
+      url: https://protocol.ai/
+  - name: Alex Potsides
+    github: achingbrain
+    affiliation:
+      name: Protocol Labs
+      url: https://protocol.ai/
+  - name: Peter Rabbitson
+    github: ribasushi
+    affiliation:
+      name: Protocol Labs
+      url: https://protocol.ai/
+  - name: Hugo Valtier
+    github: jorropo
+    affiliation:
+      name: Protocol Labs
+      url: https://protocol.ai/
+
+tags: ['architecture']
```

Comment:
TODO: I think we may want to  move this to `formats` category before this PR is merged.